databroker/src/broker.rs

```diff
@@ -1882,6 +1885,71 @@ impl<'a, 'b> AuthorizedAccess<'a, 'b> {
             )),
         }
     }
+
+    async fn validate_actuator_value(
+        &self,
+        vss_id: &i32,
+        data_value: &DataValue,
+    ) -> Result<(), (ActuationError, String)> {
+        let result_entry = self.get_entry_by_id(*vss_id).await;
+        match result_entry {
+            Ok(entry) => {
+                let validation = entry.validate_value(data_value);
+                let vss_path = entry.metadata.path;
+                match validation {
+                    Ok(_) => Ok(()),
+                    Err(UpdateError::NotFound) => {
+                        let message = format!("Could not resolve vss_path {}", vss_path);
+                        Err((ActuationError::NotFound, message))
+                    }
+                    Err(UpdateError::OutOfBounds) => {
+                        let message = format!(
+                            "Out of bounds value provided for {}: {} | Expected range [min: {}, max: {}]",
+                            vss_path,
+                            data_value,
+                            entry.metadata.min.unwrap(),
+                            entry.metadata.max.unwrap()
```

Comment:
This might lead to a crash when min or max is not specified, unfortunately some percentual paths in the vss spec only have one. e.g. Vehicle.Chassis.Axle.Row1.Wheel.Left.Brake.PadWear

Reply:
Don't put to much effort into this - as the logic might change when being returned as an "InvalidArgument" anyway.

Reply:
Done